### PR TITLE
Switch to 100col to match starfish.

### PR DESCRIPTION
**Pull request by ttung**

### CODE REVIEW

File: slicedimage/_tileset.py [hunk 1]
```diff
@@ -23,8 +29,8 @@ def add_tile(self, tile):
 
     def tiles(self, filter_fn=lambda _: True):
         """
-        Return the tiles in this tileset.  If a filter_fn is provided, only the tiles for which filter_fn returns True
-        are returned.
+        Return the tiles in this tileset.  If a filter_fn is provided, only the tiles for which "
```
Comment (ambrosejcarr) [inline slicedimage/_tileset.py]: Extra quotation mark, I think.

**Comment by ttung** [comment]
> Follow-up PR: f-style strings? :)

cannot.  this repo is python 2.7+

**Comment by ambrosejcarr** [comment]
Yuck. Ok. :-(

File: slicedimage/_tileset.py (hunk 1 above)
Comment (ttung) [reply slicedimage/_tileset.py]: Fixed.

**Comment by ttung** [comment]
@dganguli looooves python 2.7 though.

**Pull request merged**